doc/Publishing.md

```diff
@@ -116,6 +116,23 @@ For example, if you picked `0.1.0` as a version then you should run:
 
 To confirm that the tagging was correctly done, check the repo's [releases](https://github.com/eclipse-theia/theia/releases) page and confirm the release just done is listed there.
 
+## Create a release
+
+The next step of the release process is to create a new **release** on the GitHub [releases](https://github.com/eclipse-theia/theia/releases) page for the project. This will create a new release, attach the necessary assets (`.zip`, `tar.gz`, changelog link), and also notify GitHub subscribers of a new release.
+
+In order to create a new release on GitHub, one must:
+
+- navigate to the [tags](https://github.com/eclipse-theia/theia/tags) page for the project
+- select the latest tag, and click `create release` from the 'more' (`...`) toolbar item
+- update the release description to include a link to the respective section (version) in the changelog (replace the version number with the release version):
+
+   ```md
+   [Release Changelog](https://github.com/eclipse-theia/theia/blob/master/CHANGELOG.md#v120)
```

Comment:
Would it be safer to use a CHANGELOG link based on the release commit? e.g. for v1.2 the link would be: 
https://github.com/eclipse-theia/theia/blob/2aa2fa1ab091ec36ef851c4e364b322301cddb40/CHANGELOG.md

This way the link will stay good in the future, even if e.g. we chose another way to document the content of future releases.

Reply:
@marcdumais-work as of today I've been referencing the `changelog` this way (`master` at the specific release version). Do you think I should reference it this way? The only downside I see is that if we reference based on the release commit, end-users will not be able to view newer changes easily (ex: releases that are newer).

Reply:
I agree but I think that's a minor downside - it's easy enough to look at the same file on master to see all releases.